src/applications/gqrx/mainwindow.cpp

```diff
@@ -517,7 +516,7 @@ bool MainWindow::loadConfig(const QString cfgfile, bool check_crash,
         }
 
         // Update window title
-        QRegExp regexp("'([a-zA-Z0-9 \\-\\_\\/\\.\\,\\(\\)]+)'");
+        QRegExp regexp(R"('([a-zA-Z0-9 \-\_\/\.\,\(\)]+)')");
```

Comment:
When I have to update a regular expression

![image](https://user-images.githubusercontent.com/1089689/102782218-46b11600-4399-11eb-8eaa-4d52d29e9065.png)